doc/md/getting-started/getting-started.mdx

```diff
@@ -0,0 +1,548 @@
+---
+id: getting-started
+title: Quick Introduction
+sidebar_label: Quick Introduction
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Atlas CLI is an open-source tool designed to help software engineers, DBAs and DevOps
+practitioners to manage their database schemas.  Atlas users can use the [Atlas DDL](../concepts/ddl.md)
```

Comment:
i'd link to the SQL guide as its more practical. the DDL doc is a bit low level and not relevant for someonenew